less/layout.less

```diff
@@ -5,4 +5,13 @@ html, body {
 h1, h2, h3, h4, h5 {
     font-family: 'Russo One', sans-serif;
     color: @secondary-color;
+}
+.container{
+    max-width: 900px;
+}
+nav {
+    background-color: #eeeeee;
```

Comment:
We can use `@main-color` here in the LESS file since our colors are already picked out in `variables.less`